### PR TITLE
Changing FCS Wcal and Hcal x positions in Geometry file based on run 21 survey

### PR DESCRIPTION
**Pull request by akioogawa**
…21 survey

Also updating macro to store those new numbers in run22 offline DB initialization

### CODE REVIEW

File: StRoot/StFcsDbMaker/macro/fcsDetectorPosition_db.C [hunk 1]
```diff
@@ -66,6 +66,30 @@ void fcsDetectorPosition_db(char* opt = "", char* input) {
 	detpos.xoff[2]=-8.380*2.54-50; detpos.yoff[2]=0.0; detpos.zoff[2]=308.123*2.54; //x= 21.285
 	detpos.xoff[3]= 8.380*2.54+50; detpos.yoff[3]=0.0; detpos.zoff[3]=308.123*2.54;
     }
+<<<<<<< HEAD
```
Comment (klendathu2k) [inline StRoot/StFcsDbMaker/macro/fcsDetectorPosition_db.C]: It looks like you might need to merge this by hand and recommit?

**Comment by akioogawa** [comment]
I looked up "failed to solve: failed to read dockerfile: failed to fetch remote https://github.com/star-bnl/star-sw.git: exit status 128" problem. But I have no idea. Looks like an issue with Docker configuration issue rather than code issue.

**Comment by plexoos** [comment]
Yes, it does look like a hiccup on the GitHub side. I restarted the jobs

**Comment by klendathu2k** [comment]
I think that, pending successful build, we could merge.  @akioogawa  But, one of the _macros_ has merge conflicts.  Would prefer a commit that resolves that issue.

File: StRoot/StFcsDbMaker/macro/fcsDetectorPosition_db.C [hunk 2]
```diff
@@ -66,6 +66,30 @@ void fcsDetectorPosition_db(char* opt = "", char* input) {
...
+=======
```
Comment (plexoos) [inline StRoot/StFcsDbMaker/macro/fcsDetectorPosition_db.C]:
```suggestion
```

File: StRoot/StFcsDbMaker/macro/fcsDetectorPosition_db.C [hunk 3]
```diff
@@ -66,6 +66,30 @@ void fcsDetectorPosition_db(char* opt = "", char* input) {
...
+	detpos.xoff[3]= 8.380*2.54; detpos.yoff[3]=0.0; detpos.zoff[3]=308.123*2.54;
+    }
+    else if(data.Contains("run22ofl")){  //closed
+	storeTime = "2021-10-25 00:00:00"; flavor="ofl";
+        detpos.xoff[0]=-6.850*2.54; detpos.yoff[0]=0.0; detpos.zoff[0]=279.592*2.54; //x= 17.399
+	detpos.xoff[1]= 6.850*2.54; detpos.yoff[1]=0.0; detpos.zoff[1]=279.592*2.54;
+	detpos.xoff[2]=-8.380*2.54; detpos.yoff[2]=0.0; detpos.zoff[2]=308.123*2.54; //x= 21.285
+	detpos.xoff[3]= 8.380*2.54; detpos.yoff[3]=0.0; detpos.zoff[3]=308.123*2.54;
+    }
+>>>>>>> 1bb48ab955... Changing FCS Wcal and Hcal x positions in Geometry file beased on run21 survey
```
Comment (plexoos) [inline StRoot/StFcsDbMaker/macro/fcsDetectorPosition_db.C]:
```suggestion
```

File: StRoot/StFcsDbMaker/macro/fcsDetectorPosition_db.C [hunk 4]
```diff
@@ -66,6 +66,30 @@ void fcsDetectorPosition_db(char* opt = "", char* input) {
...
 	detpos.xoff[3]= 8.380*2.54+50; detpos.yoff[3]=0.0; detpos.zoff[3]=308.123*2.54;
     }
+<<<<<<< HEAD
+=======
+    else if(data.Contains("run21Jun27")){ //closed
+        storeTime = "2021-06-27 15:00:00"; flavor="ofl";
+        detpos.xoff[0]=-6.850*2.54; detpos.yoff[0]=0.0; detpos.zoff[0]=279.592*2.54; //x= 17.399
+	detpos.xoff[1]= 6.850*2.54; detpos.yoff[1]=0.0; detpos.zoff[1]=279.592*2.54;
+	detpos.xoff[2]=-8.380*2.54; detpos.yoff[2]=0.0; detpos.zoff[2]=308.123*2.54; //x= 21.285
+	detpos.xoff[3]= 8.380*2.54; detpos.yoff[3]=0.0; detpos.zoff[3]=308.123*2.54;
```
Comment (plexoos) [inline StRoot/StFcsDbMaker/macro/fcsDetectorPosition_db.C:74]: Fix indentation here and below

**Review by klendathu2k — APPROVED** [review]
Looks good to me.

File: StRoot/StFcsDbMaker/macro/fcsDetectorPosition_db.C (hunk 1 above)
Comment (plexoos) [reply StRoot/StFcsDbMaker/macro/fcsDetectorPosition_db.C]: Hm... No, it looks as Akio has already removed those lines and it is good to go.

**Review by plexoos — APPROVED** [review]

File: StRoot/StFcsDbMaker/macro/fcsDetectorPosition_db.C (hunk 1 above)
Comment (akioogawa) [reply StRoot/StFcsDbMaker/macro/fcsDetectorPosition_db.C]: Well? What do I need?

**Comment by plexoos** [comment]
@jdbrice?

**Comment by klendathu2k** [comment]
@jdbrice Daniel, could you take a look at this code change and review?  (You're marked as a code owner).

**Review by jdbrice — APPROVED** [review]
Looks good

**Pull request merged**